docs/source/using/addons.adoc

```diff
@@ -112,6 +112,23 @@ The following table shows the supported variables that are available for interpo
 |Name of the current add-on.
 |===
 
+The commands xref:../command-ref/minishift_start.adoc#[`minishift start`] as well as xref:../command-ref/minishift_addons_apply.adoc#[`minishift addons apply`] also provide a `--addon-env` flag which allow to pass variables for interpolation dynamically, for example:
+
+----
+$ minishift addons apply --addon-env ACME_TOKEN=1234 acme
+----
+
+The `--addon-env` flag can be specified multiple times to define more than one variables for interpolation.
```

Comment:
You should mention the short form as well i.e. -a . Also we should mention about how it can take variables from environment.